docs/src/main/asciidoc/security-customization.adoc

```diff
@@ -241,37 +241,63 @@ The solution is to activate the request context, the following example shows how
 import io.quarkus.security.identity.AuthenticationRequestContext;
 import io.quarkus.security.identity.SecurityIdentity;
 import io.quarkus.security.identity.SecurityIdentityAugmentor;
-import io.quarkus.security.runtime.QuarkusSecurityIdentity;
 import io.smallrye.mutiny.Uni;
 
 import javax.enterprise.context.ApplicationScoped;
-import javax.enterprise.context.control.ActivateRequestContext;
-import java.util.function.Supplier;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
 
 @ApplicationScoped
 public class RolesAugmentor implements SecurityIdentityAugmentor {
+
+    @Inject
+    Instance<SecurityIdentitySupplier> identitySupplierInstance;
+
     @Override
     public Uni<SecurityIdentity> augment(SecurityIdentity identity, AuthenticationRequestContext context) {
         if(identity.isAnonymous()) {
             return Uni.createFrom().item(identity);
         }
 
-        // Hibernate ORM is blocking
-        return context.runBlocking(build(identity));
+        // Hibernate ORM is
```

Comment:
@vanny96 Can you please complete this comment ?

Reply:
Whoopsie, should be addressed!